src/prefixmaps/data/bioregistry.csv

```diff
@@ -126,23 +120,16 @@ bioregistry,BSPO,http://purl.obolibrary.org/obo/BSPO_,canonical
 bioregistry,BTO,http://purl.obolibrary.org/obo/BTO_,canonical
 bioregistry,bugbase.expt,http://identifiers.org/bugbase.expt/,canonical
 bioregistry,bugbase.protocol,http://identifiers.org/bugbase.protocol/,canonical
-bioregistry,bykdb,http://identifiers.org/bykdb/,canonical
 bioregistry,C4O,http://purl.org/spar/c4o/,canonical
-bioregistry,cabri,http://identifiers.org/cabri/,canonical
-bioregistry,cadsr,http://identifiers.org/cadsr/,canonical
 bioregistry,caloha,https://biopragmatics.github.io/providers/caloha/,canonical
 bioregistry,cameo,http://identifiers.org/cameo/,canonical
 bioregistry,caps,http://identifiers.org/caps/,canonical
 bioregistry,CARO,http://purl.obolibrary.org/obo/CARO_,canonical
 bioregistry,cas,http://identifiers.org/cas/,canonical
 bioregistry,cath,http://identifiers.org/cath/,canonical
 bioregistry,cath.domain,http://identifiers.org/cath.domain/,canonical
-bioregistry,cath.superfamily,http://identifiers.org/cath.superfamily/,canonical

```

Comment:
is there a reason this is disappearing?

Reply:
Its URI prefix is redundant of `cath` (though this is obscured by using identifiers.org URIs)

Reply:
I see. I find the subspacing decisions (inherited from identifiers.org) confusing and inconsistent. Sometimes the top level corresponds to a subspace, sometimes it is its own thing. But we can take this to the existing ticket: https://github.com/biopragmatics/bioregistry/issues/133

I don't love prefixes or URLs disappearing, we should have guards against this in future but I think some churn in the initial phase is acceptable.